scripts/playground/templates/airflow.yaml

```diff
@@ -62,13 +70,6 @@ spec:
           replicas: 1
           defaultUser:
             password: $password
-        migrateDatabaseJob:
```

Comment:
Do we not need this `migrateDatabaseJob` anymore?

Reply:
we do but the we do not need to mess with the job annotations directly anymore since the upstream chart added the `migrateDatabaseJob.useHelmHooks` field.

Reply:
I moved up the block referencing the new value to the top because it's actually very important we disable helm hooks since we deploy via Argo (Argo treats helm hooks slightly differently than helm does and causes initial installs to hang)